yarn.lock

```diff
@@ -2371,6 +2381,15 @@
     "@sentry/utils" "7.56.0"
     tslib "^1.9.3"
 
+"@sentry/core@7.57.0-beta.0":
```

Comment:
you'll want to run `npx yarn-deduplicate && yarn install` when you see a bunch of duplicate packages

Reply:
I did do that, the problem is that beta versions do not satisfy the version range of transitive dependencies e.g. of sentry-cli (so @sentry/core@7.57.0-beta.0 does not satisfy @sentry/core@^7.19.0, for example). I have added yarn resolutions to force the transitive dependencies for now, once we update back to the regular track we can remove these again!